src/overlay/Peer.cpp

```diff
@@ -852,7 +852,13 @@ Peer::recvMessage(StellarMessage const& stellarMsg)
             return;
         }
 
-        mFlowControlState = Peer::FlowControlState::ENABLED;
+        if (mFlowControlState != Peer::FlowControlState::ENABLED)
```

Comment:
not following why this if condition is here

Reply:
Removed.